src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -2849,22 +2849,26 @@ namespace RuntimeLanguageProcessor {
 							ShowContinueError( "Blank " + cAlphaFieldNames( 1 ) );
 							ShowContinueError( "Blank entry will be skipped, and the simulation continues" );
 						} else if ( ! errFlag ) {
-							VariableNum = FindEMSVariable( cAlphaArgs( ErlVarLoop ), 0 );
 							// Still need to check for conflicts with program and function names too
 
-							if ( VariableNum > 0 ) {
-								ShowSevereError( RoutineName + cCurrentModuleObject + ", invalid entry." );
-								ShowContinueError( "Invalid " + cAlphaFieldNames( ErlVarLoop ) + '=' + cAlphaArgs( ErlVarLoop ) );
-								ShowContinueError( "Name conflicts with an existing global variable name" );
-								ErrorsFound = true;
+							if ((cCurrentModuleObject.compare("ExternalInterface:FunctionalMockupUnitImport:To:Variable") == 0) && ErlVarLoop > 1){
+								// Only validate first field of object ExternalInterface:FunctionalMockupUnitImport:To:Variable. Skip others
 							} else {
-								VariableNum = NewEMSVariable( cAlphaArgs( ErlVarLoop ), 0 );
-								if ( GlobalNum > NumUserGlobalVariables ) {
-									// Initialize variables for the ExternalInterface variables.
-									// This object requires an initial value.
-									ExternalInterfaceInitializeErlVariable( VariableNum, SetErlValueNumber( rNumericArgs( 1 ) ), false );
-								}
+								VariableNum = FindEMSVariable( cAlphaArgs( ErlVarLoop ), 0 );
```

Comment:
Empty `if` blocks that are there just to coerce the code path to the other side are kinda funny.  Could we just change the if block to the opposite of what it finds now so that the empty section will be implied?  Or if we are in a loop here could we just `continue`?